app/controllers/groups/self_inscription_controller.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Pfadibewegung Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class Groups::SelfInscriptionController < CrudController
```

Comment:
Eigentlich ist es Konvention in Rails, Controller-Namen im Plural zu schreiben: SelfInscriptionsController. Aber ich sehe dass es beim SelfRegistrationController auch schon falsch gemacht wurde...

Reply:
Wenn man beide anpassen will (Registration und Inscription) sind recht viele Files betroffen. Bei den Migrations wusste ich nicht, ob ich den Namen verändern darf. Bei der Gruppe braucht es vermutlich auch eine Migration, wenn man das anpassen will. Für heute war das etwas overkill. Das sollte man eher unabhängig von diesem Feature machen.